components/pokemonapp.brs

```diff
@@ -1,9 +1,13 @@
 function init()
     m.focusedItemIndex = 1
     m.firstChild = 0
+    m.focusedPokemonIndex = 0
     m.descriptionLabel = m.top.FindNode("descriptionLabel")
+    m.ratingLabel = m.top.FindNode("ratingLabel")
     m.titleLabel = m.top.FindNode("titleLabel")
     m.pokeList = m.top.FindNode("PokemonRowList")
+    m.pokeRatingList = m.top.FindNode("StarsList")
+    m.registry = CreateObject("roRegistrySection","PokemonAppRegistry")
```

Comment:
You can declare it only when you use it.
Don't keep it global because you don't use it in multiple places inside the component